src/main/kotlin/Interfaces.kt

```diff
@@ -73,28 +73,32 @@ open class Car(
 }
 
 
-
 class Dalorian(
 
-): Car(), TimeTravelable {
+) : Car(), TimeTravelable {
 
     override var charged: Boolean = false
 
     override fun chargeFluxCapacitor() {
-        if (charged == false) {
+        if (!charged) {
             charged = true
             println("You refill the Flux Capacitor with plutonium and set the meter to 1.21 gigawatts, a few moments later it is fully charged")
         } else {
             println("Flux Capacitor is already fully charged")
         }
     }
 
+    override fun inspectFluxCapacitor() {
+        if (charged) println("Flux capacitor... fluxing") else println("Let's fill up Mr. Fusion!")
+    }
+
```

Comment:
I added the above so that you could see an optimal use of an `if... else` statement with single line codeblocks. When you have these types of if-else statements trying to get them on the single line makes them a bit more readable. Your other two functions are close, baring the flip of `charged` so it might be worth thinking on how to achieve something similar there.

Reply:
Similar to ternary operator in other languages, I like this